src/workflow/elements/Element.js

```diff
@@ -0,0 +1,7 @@
+const Joi = require('joi')
+
+const ELEMENT_BASE_SCHEMA = Joi.object({ id: Joi.string().required() })
```

Comment:
The Id of every element is now a uuid! This makes the generation of unique IDs very comfortable.